ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<ul><li>If we're listing the name in the notebook anyway, do we need to instruct users to record the name? Can we remove these instructions? (same comment for datasets/models)</li></ul><p> </p><p> </p><blockquote>To get a reference to the model that you just deployed, you check the <code>display_name</code> of each model to the models deployed to each endpoint.</blockquote><ul><li>If I understood this correctly, suggest: "...just deployed, check the display_name of each model against the models that are deployed to each endpoint."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>First comment: Ack. I think it's helpful to let customers know that they can either: a) copy the endpoint name when the SDK prints it out; or b) get a list of all their endpoints and select the one they need from the list. Same for datasets and models.</p>

Reply:
<p>Second comment: Done.</p>